services/vod-to-live/service/src/domains/validation/utils.ts

```diff
@@ -0,0 +1,67 @@
+import { DetailedVideo } from '@axinom/mosaic-messages';
+import { WebhookValidationMessage } from '@axinom/mosaic-service-common';
+import { Config, HOUR_IN_SECONDS } from '../../common';
+import { getPlaylistDurationInSeconds } from '../utils';
+
+export interface ValidationResult {
+  errors: WebhookValidationMessage[];
+  warnings: WebhookValidationMessage[];
+}
+export const validateVideo = (
+  video: DetailedVideo,
+  config: Config,
+): ValidationResult => {
+  const validationResult: ValidationResult = {
+    errors: [],
+    warnings: [],
+  };
+  if (video.video_encoding.is_protected) {
+    if (config.isDrmEnabled) {
+      if (video.video_encoding.video_streams.find((s) => !s.key_id)) {
```

Comment:
Please adjust this not to include Subtitle and Caption streams, they always have null key IDs. And cover this case with unit test(s).